supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/GenomicsFactory.scala

```diff
@@ -200,16 +200,13 @@ case class GenomicsFactory(applicationName: String, authMode: GoogleAuthMode, en
       val adjustedBootDiskSize = {
         val fromRuntimeAttributes = createPipelineParameters.runtimeAttributes.bootDiskSize
         // Compute the decompressed size based on the information available
-        val actualSizeInBytes = createPipelineParameters.jobDescriptor.dockerSize.map(_.toFullSize(DockerConfiguration.instance.sizeCompressionFactor)).getOrElse(0L)
-        val actualSizeInGB = MemorySize(actualSizeInBytes.toDouble, MemoryUnit.Bytes).to(MemoryUnit.GB).amount
-        val actualSizeRoundedUpInGB = actualSizeInGB.ceil.toInt
+        val userCommandImageSizeInBytes = createPipelineParameters.jobDescriptor.dockerSize.map(_.toFullSize(DockerConfiguration.instance.sizeCompressionFactor)).getOrElse(0L)
+        val userCommandImageSizeInGB = MemorySize(userCommandImageSizeInBytes.toDouble, MemoryUnit.Bytes).to(MemoryUnit.GB).amount
+        val userCommandImageSizeRoundedUpInGB = userCommandImageSizeInGB.ceil.toInt
 
-        // If the size of the image is larger than what is in the runtime attributes, adjust it to uncompressed size
-        if (actualSizeRoundedUpInGB > fromRuntimeAttributes) {
-          jobLogger.info(s"Adjusting boot disk size to $actualSizeRoundedUpInGB GB to account for docker image size")
-        }
-
-        Math.max(fromRuntimeAttributes, actualSizeRoundedUpInGB) + ActionUtils.cromwellImagesSizeRoundedUpInGB
+        val totalSize = fromRuntimeAttributes + userCommandImageSizeRoundedUpInGB + ActionUtils.cromwellImagesSizeRoundedUpInGB
```

Comment:
Basically we're running into cases where that arithmetic doesn't work, even with greatly exaggerated compression factors for Docker images. I suspect the current system breaks down because there are other things we don't control being installed on the boot disk besides Docker images and they take up enough space to cause problems. 

This new code uses a more conservative model that assumes the `fromRuntimeAttributes` number is exclusive of Docker images. Realistic compression factors are used to multiply the compressed image sizes, which are then added to `fromRuntimeAttributes`.